### PR TITLE
All New FHIR Location Resource 

### PR DESCRIPTION
**Pull request by yashrajbothra**
#### Short description of what this resolves:
Added new endpoints for location resource

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by yashrajbothra** [comment]
Hii @bradymiller new Uuid Mechanism is plugged and testing well :)

File: src/Services/LocationService.php [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        $isValid = BaseValidator::validateIdMultiple(
+            "uuid",
+            array(
+                self::FACILITY_TABLE,
+                self::PRACTITIONER_TABLE,
+                self::PATIENT_TABLE
+            ),
+            $uuid,
+            true
+        );
```
Comment (bradymiller) [inline src/Services/LocationService.php:135]: Is the validating the correct table(s). Aren't the uuid's that you are searching with stored in the uuid_mapping table?

Comment (yashrajbothra) [reply src/Services/LocationService.php:135]: Yup, Also we don't need this Multiple Id function now :)

**Comment by bradymiller** [comment]
hi @yashrajbothra , Code looks good and sounds like testing well for you, so bringing this in. Thanks for yet another fhir endpoint! -brady

**Pull request merged**